### PR TITLE
Translate specific terms in readme.md

### PR DESCRIPTION
**Pull request by soyvikman**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by soyvikman** [comment]
I translate some specific terms of the english version of aritmetic operators!

File: guide/spanish/javascript/arithmetic-operation/index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ JavaScript proporciona al usuario cinco operadores aritméticos: `+` , `-` , `*`
...
 ```
-2 + 3          // returns 5 
- true + 2       // interprets true as 1 and returns 3 
- false + 5      // interprets false as 0 and returns 5 
- true + "bar"   // concatenates the boolean value and returns "truebar" 
- 5 + "foo"      // concatenates the string and the number and returns "5foo" 
- "foo" + "bar"  // concatenates the strings and returns "foobar" 
+2 + 3          // retorna 5 
+ true + 2       // interpreta true como 1 y retorna 3 
+ false + 5      // interpreta false como y retorna 5
```
Comment (hestellez) [inline guide/spanish/javascript/arithmetic-operation/index.md]: Should be:
//interpreta false como 0 y retorna 5

File: guide/spanish/javascript/arithmetic-operation/index.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ JavaScript proporciona al usuario cinco operadores aritméticos: `+` , `-` , `*`
...
- true + 2       // interprets true as 1 and returns 3 
- false + 5      // interprets false as 0 and returns 5 
- true + "bar"   // concatenates the boolean value and returns "truebar" 
- 5 + "foo"      // concatenates the string and the number and returns "5foo" 
- "foo" + "bar"  // concatenates the strings and returns "foobar" 
+2 + 3          // retorna 5 
+ true + 2       // interpreta true como 1 y retorna 3 
+ false + 5      // interpreta false como y retorna 5 
+ true + "bar"   // concatena el valor booleano y retorna "truebar" 
+ 5 + "foo"      // concatena la cadena y el numero y retorna "5foo"
```
Comment (hestellez) [inline guide/spanish/javascript/arithmetic-operation/index.md]: should be "número" with an accent mark

**Review by hestellez — CHANGES_REQUESTED** [review]

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**